libdnf/dnf-module.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc.
+ *
+ * Licensed under the GNU Lesser General Public License Version 2.1
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301 USA
+ */
+
+#ifndef __DNF_MODULE_H
+#define __DNF_MODULE_H
+
+#include <vector>
+
+#include "dnf-types.h"
+#include "hy-query.h"
+#include "module/ModulePackageContainer.hpp"
+#include "sack/query.hpp"
+
+namespace libdnf {
+
+class ModuleExceptionList : public std::exception {
+public:
+    explicit ModuleExceptionList() {}
+    const std::vector<ModulePackageContainer::Exception> & list() const noexcept { return _list; }
+    void add(const ModulePackageContainer::Exception &what) { _list.push_back(what); }
+    void add(const std::string &what) { _list.push_back(ModulePackageContainer::Exception(what)); }
+    bool empty() { return _list.empty(); }
+
+private:
+    std::vector<ModulePackageContainer::Exception> _list;
+};
+
+bool dnf_module_dummy(const std::vector<std::string> & module_list);
```

Comment:
Please can you add a description. I am not sure what it does

Reply:
`dnf_module_dummy` was inherited from PR #508 -- upon which this PR is based. I does nothing useful and was included as placeholder when creating the files and tests while implementing the modular API calls. It can be safely ignored and should be discarded. I'll create another commit to do just that.